### PR TITLE
python 3.8 external pins pandas<1.1.0, numpy<1.19.0

### PR DESCRIPTION
**Pull request by dunckerr**
New external pins.  Note this branch was branched off feature/python-3.8

### CODE REVIEW

**Comment by vietlq** [comment]
Just curious if the versions are aligned with Pegagus?

**Comment by dunckerr** [comment]
hi @vietlq yes these pins align with pegasus as discussed on slack.  Im planning on merging this into master soonish - Im just checking my changes first.

**Review by aflag — APPROVED** [review]

File: arctic/date/_util.py [hunk 1]
```diff
@@ -2,8 +2,7 @@
 import datetime
 import sys
 from datetime import timedelta
-
-import pandas as pd
+import pandas
```
Comment (mehertz) [inline arctic/date/_util.py:5]: I'd bet the rest of the codebase will import it as `pd` - why change this just for this file?

**Comment by dunckerr** [comment]
Hi Viet - yes I got the new external pins from the Pegasus team.  I just need to check a couple of pandas changes I had to make then I’ll get this merged.
Cheers
Duncan

On Apr 10, 2022, at 9:35 PM, Viet ***@***.***> wrote:

﻿

Just curious if the versions are aligned with Pegagus?

**Pull request merged**